### ISSUE TITLE
Py 3.9 failures: 'xml.etree.ElementTree.Element' object has no attribute 'getchildren'

### ISSUE THREAD

**Issue by QuLogic** [jbarlow83/OCRmyPDF#583]
When running on Python 3.9, tests fail with the above error.

```pytb
Traceback (most recent call last):
  File "/usr/lib64/python3.9/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/builddir/build/BUILDROOT/ocrmypdf-10.1.0-1.fc33.x86_64/usr/lib/python3.9/site-packages/ocrmypdf/_sync.py", line 214, in exec_page_sync
    ocr_out = render_hocr_page(hocr_out, page_context)
  File "/builddir/build/BUILDROOT/ocrmypdf-10.1.0-1.fc33.x86_64/usr/lib/python3.9/site-packages/ocrmypdf/_pipeline.py", line 612, in render_hocr_page
    hocrtransform.to_pdf(
  File "/builddir/build/BUILDROOT/ocrmypdf-10.1.0-1.fc33.x86_64/usr/lib/python3.9/site-packages/ocrmypdf/hocrtransform.py", line 191, in to_pdf
    elemtxt = self._get_element_text(elem).rstrip()
  File "/builddir/build/BUILDROOT/ocrmypdf-10.1.0-1.fc33.x86_64/usr/lib/python3.9/site-packages/ocrmypdf/hocrtransform.py", line 112, in _get_element_text
    for child in element.getchildren():
AttributeError: 'xml.etree.ElementTree.Element' object has no attribute 'getchildren'
```
There are about 15 failures, but I think they all end in this same traceback.

**To Reproduce**
```bash
pytest -ra -n auto
```

Here is a [build log for 9.8.2](https://koji.fedoraproject.org/koji/taskinfo?taskID=45997958), but it also fails locally for me with 10.1.0.

**System**
 - OS: Fedora Rawhide
 - OCRmyPDF Version: 9.8.2/10.1.0
 - How did you install ocrmypdf? Not installed; building a package.

**Comment by QuLogic** [comment]
Note, this method was deprecated since 3.2, though only warned in 3.8: https://docs.python.org/3.8/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.getchildren

**Comment by jbarlow83** [comment]
Thanks - fixed in v10.2.0 along with other issues in that module.
